features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/TreeView/PermissionsSelector.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Let's add the licence header to this file.

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/9045/commits/b857d16f62df6be3152b273bd00333653436d8b0